src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -105,7 +112,7 @@ function ProductGallery({ title, slug }: Props) {
             />
           ))
         ) : (
-          <div className="product-listing__data-loading">Loading…</div>
+          <ProductGrid page={0} pageSize={0} products={[]} />
```

Comment:
#328 removes the loading `div`. Maybe we can wait until the merge to think of a better way to deal with it.